openlibrary/macros/ShareModal.html

```diff
@@ -49,6 +49,20 @@
               </a>
               <div class="share-source">$_('Embed')</div>
             </li>
+
+            <li>
+              <a class="copy-url-btn" title="$_('Copy url to clipboard')" data-ol-link-track="Share|CopyURL"
+                  onclick="navigator.clipboard.writeText('openlibrary.org'+'$page_url');">
```

Comment:
```suggestion
              <a class="copy-url-btn" title="$_('Copy url to clipboard')" data-ol-link-track="Share|CopyURL">
```
Move inline JS to Webpack.  To do so, you can update [this function](https://github.com/internetarchive/openlibrary/blob/8ca25d920acf98ea431f0fae1e9e7876339315aa/openlibrary/plugins/openlibrary/js/modals/index.js#L5-L7) to do the following:

1. Get a reference to the "Copy URL" icon.  This query should work `#social-modal-content .copy-url-btn`.
2. Add a click listener to the icon.

The click listener should:
1. Copy the URL to the clipboard.
2. Close the modal. (Running `$.colorbox.close()` will achieve this)
3. Display a toast message that tells the patron that the message has been copied to their clipboard.  You can use [this function](https://github.com/internetarchive/openlibrary/blob/8ca25d920acf98ea431f0fae1e9e7876339315aa/openlibrary/plugins/openlibrary/js/modals/index.js#L172) to display the toast message (only pass `message` --- `$parent` is not needed).